widgetastic_manageiq/__init__.py

```diff
@@ -2651,12 +2651,15 @@ class ListView(EntitiesConditionalView):
 
         @property
         def entity_names(self):
-            """ looks for entities and extracts their names
+            """ looks for entities and extracts their names or keys
 
             Returns: all current page entities
             """
             try:
-                return [row.name.text for row in self.elements.rows()]
+                try:
```

Comment:
I don't think a `try` inside `try` in required, you can simply write multiple `except` and catch accordingly